spec.bs

```diff
@@ -1563,8 +1758,8 @@ an <a spec="turtledove">auction config</a> |auctionConfig| and a
             1. [=Append an entry to the contribution cache|Append=] |entry| to
                 the [=contribution cache=].
 
-1. [=list/iterate|For each=] |origin| → |batchingScope| of
-    |auctionConfig|'s [=auction config/batching scope map=]:
+1. [=list/iterate|For each=] (|origin|, |aggregationCoordinator|) →
+    |batchingScope| of |auctionConfig|'s [=auction config/batching scope map=]:
```

Comment:
where is |auctionConfig|'s [=auction config/batching scope map=] set?

Reply:
Entries are inserted here: https://pr-preview.s3.amazonaws.com/patcg-individual-drafts/private-aggregation-api/pull/106.html#ref-for-auction-config-batching-scope-map

Reply:
Ack.